versions.sh

```diff
@@ -81,7 +81,7 @@ for version in "${versions[@]}"; do
 	# order here controls the order of the library/ file
 	for suite in \
 		bookworm \
-		bullseye \
+		trixie \
```

Comment:
Order is actually load-bearing here, so `trixie` should be listed first. :eyes:

Reply:
I've swapped the two version